src/babelPluginTsdx.ts

```diff
@@ -10,7 +10,10 @@ export const isTruthy = (obj?: any) => {
   return obj.constructor !== Object || Object.keys(obj).length > 0;
 };
 
-const replacements = [{ original: 'lodash', replacement: 'lodash-es' }];
+const replacements = [
+  { original: 'lodash', replacement: 'lodash-es' },
+  { original: 'lodash-es/fp', replacement: 'lodash/fp' },
```

Comment:
does this actually work? seems very hacky and would rely on the plugin using a serial two-pass replace. In any case, we'd want to add a test for this

Reply:
It works, but I agree your solution seems better. 👍